nautobot_device_lifecycle_mgmt/templates/nautobot_device_lifecycle_mgmt/softwareimagelcm_list.html

```diff
@@ -0,0 +1,3 @@
+{% extends 'generic/object_list.html' %}
```

Comment:
do we need this custom template? can't we use the default one provided by Nautobot?

Reply:
This is needed to change plural display name for the model. The alternative is to specify `verbose_name_plural` in the `Meta` class of the model. I don't like changing `verbose_name_plural` because that affects the naming of the GraphQL endpoints for the model. I find extending the template to be more explicit and having less side effects.